nflow-rest-api-common/src/main/java/io/nflow/rest/v1/converter/ListWorkflowInstanceConverter.java

```diff
@@ -50,6 +51,7 @@ public ListWorkflowInstanceResponse convert(WorkflowInstance instance, Set<Workf
     resp.started = instance.started;
     resp.retries = instance.retries;
     resp.signal = instance.signal.orElse(null);
+    resp.isArchived = instance.isArchived ? TRUE : null;
```

Comment:
I would put both true and false values to the response, and only use null when archived instances are not queried at all (then the value can be omitted).